<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingWindow</class>
 <widget class="QMainWindow" name="TestingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="weeks">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="game">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>9</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="team">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>9</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="pos">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>9</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="player">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>9</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="season">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>9</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="week_3">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>9</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Week</string>
    </property>
   </widget>
   <widget class="QLabel" name="week">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>9</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Live">
    <property name="geometry">
     <rect>
      <x>638</x>
      <y>9</y>
      <width>39</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Live</string>
    </property>
   </widget>
   <widget class="QLabel" name="globalstate">
    <property name="geometry">
     <rect>
      <x>719</x>
      <y>9</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No State</string>
    </property>
   </widget>
   <widget class="QPushButton" name="claimname">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>35</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Claim Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FantassyNameIn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>36</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="fantasynames">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>36</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Clone">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>35</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beoracle">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>64</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Be Oracle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nmeonic">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>64</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test NM</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rundataagent">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>67</y>
      <width>44</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="projection">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>65</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="projectit">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>64</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Project</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetGameResult">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>93</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Results</string>
    </property>
   </widget>
   <widget class="QComboBox" name="gameID">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>94</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="allgames">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>96</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> All games</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetTx">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>93</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GetTx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updatelb">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>93</y>
      <width>104</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>UpdateLeaderBoard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="result">
    <property name="geometry">
     <rect>
      <x>638</x>
      <y>93</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RESULT</string>
    </property>
   </widget>
   <widget class="QListWidget" name="staging_data">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>122</y>
      <width>237</width>
      <height>299</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="staging_tx">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>122</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="awards">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>122</y>
      <width>225</width>
      <height>299</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="transitions">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>428</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StageBlock">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>427</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>StageBlock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendBlock">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>427</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SendBlock</string>
    </property>
   </widget>
     <widget class="QPushButton" name="stage_player">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>427</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
       <string>Stage PLayer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="out">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>457</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="MsgButton">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>456</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3" colspan="2">
     <widget class="QPushButton" name="commit_player">
      <property name="text">
       <string>Commit Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Update_PLayers_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>hackIt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
